netbox/utilities/forms.py

```diff
@@ -530,6 +530,8 @@ def __init__(self, *args, **kwargs):
                 """
 
     def to_python(self, value):
+        if value is None:
+            return list()
```

Comment:
Was something throwing an error?  I don't quite see what this is for.

Reply:
`ExpandableNameField` always expected a value. Since `label` accepts a blank, this failed. I had to fix it so it would return an empty list if the value was `None`